internal/clients/diffreviewer/circleci/circleci_service.go

```diff
@@ -111,6 +177,68 @@ func (s *Service) GetDiff() ([]*diffreviewer.FileDiff, error) {
 
 // WriteComments posts the findings as annotations to the github check
 func (s *Service) WriteComments(comments []*diffreviewer.Comment) error {
-	//TODO: implement
-	return nil
+	if len(comments) == 0 {
+		s.Logger.Debug("no sensitive items found")
+		return nil
+	}
+	if s.PrDetails.PrNumber != nil {
+		githubComments := s.createGithubPullRequestComments(comments)
+		for _, c := range githubComments {
+			_, _, err := s.GithubClient.PullRequestsService().CreateComment(
+				context.Background(),
+				s.PrDetails.Owner,
+				s.PrDetails.Repo,
+				*s.PrDetails.PrNumber,
+				c,
+			)
+			if err != nil {
+				s.Logger.Error(fmt.Sprintf("Error writing comment to pull request: %s", err.Error()))
+				s.Logger.Error(*c.Body)
```

Comment:
Github unfortunately has a rate limit of 5000 requests per hour per user (total requests made by all of a user's tokens <= 5000), so I am logging the finding comment in the circle workflow if the rate limit is exceeded.
https://developer.github.com/v3/#rate-limiting